src/h265_nal_unit_parser.cc

```diff
@@ -348,6 +347,7 @@ void H265NalUnitPayloadParser::NalUnitPayloadState::fdump(
     case RASL_N:
     case RASL_R:
       slice_segment_layer.fdump(outfp, indent_level);
+      [[fallthrough]];
```

Comment:
You're overthinking this: I just forgot the `break;` (and the linter didn't complain). Same in the other one

Reply:
Not really overthinking - if not adding break or fallthrough the build would fail. Sounds like this should be changed to break ?

Reply:
> if not adding break

Yeah, that's what I meant by "overthinking" when I saw the `[[fallthrough]];`. I just forgot the "break" :). Please add them here and there